tests/test_policy.py

```diff
@@ -225,6 +224,7 @@ def test_cem_based(algo):
     }
     agent = omnisafe.Agent(algo, env_id, custom_cfgs=custom_cfgs)
     agent.learn()
+    agent.render()
```

Comment:
As model-based algorithms have many branches of evaluation, I render all of them to make a trust-worthy test.